content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -159,6 +159,20 @@ the NetworkPolicy acts on may be the IP of a `LoadBalancer` or of the Pod's node
 For egress, this means that connections from pods to `Service` IPs that get rewritten to
 cluster-external IPs may or may not be subject to `ipBlock`-based policies.
 
+__[]__: The 'empty map' matches all traffic from/to <Any>, and is intended to permit all traffic.
```

Comment:
`<Any>` looks like a coding generic, but on the web it just gets swallowed up In the preview the `<Any>` doesn't render.

```suggestion
You can use an empty YAML map (`[]`) when you want to match all addresses. In a JSON
manifest, use an empty object (`{}`).
```

Alternatively, rather than relying on coercion of an empty YAML array to an empty YAML map, how about using `{}` in the example even for YAML? That works, right?

Reply:
+1

Reply:
Yes, I think the author is referring to the `Any` type for serialization/deserialization of data and the YAML map for representing this data.
If you surround the `<Any>` with backquotes, I suspect the text will render.
The suggested text seems to work as well.

Reply:
It's not a map, it's an empty array. `from`/`to` take an array of `NetworkPolicyPeer`s, so if you want to specify "nothing" here, you have to say `[]`.

But the rest of this section is describing the values that individual `NetworkPolicyPeer`s can have, while `[]` is a value that `to`/`from` can have _instead_ of having an array of `NetworkPolicyPeer`.

Additionally, it is unnecessary, since matching everything is the default when you don't specify any `to`/`from`, so the example below could just remove the `from: []` line and it would still mean the same thing (and is at least equally clear in meaning, if not more clear).

So maybe adding an example of matching everything would be good (we could definitely use more examples), but I think it should be specified in the standard way, rather than with `[]`.